### ISSUE TITLE
Dangerous setup command

### ISSUE THREAD

**Issue by pavlm** [BytemarkHosting/docker-webdav#29]
Host system file permissions easily can be destroyed when system folder mapped to webdav.
This line is dangerous: `chown -R www-data:www-data "/var/lib/dav"`

**Comment by ghost** [comment]
Yes,my debian server's host folder had been changed when i use this docker image.

**Comment by jeff47** [comment]
Not just dangerous... lazy design that severely limits the utility.  I suggest trying sftpgo as a robust alternative that doesn't have this issue.
